src/libsystem_networkextension/CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+project(system_networkextension)
+
+cmake_minimum_required(VERSION 2.4.0)
+
+add_definitions(-DTARGET_OS_MAC=1)
+add_definitions(-D__APPLE__ -D__DYNAMIC__)
+
+set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -nostdinc -D__DARWIN_UNIX03 -fPIC -w")
+
+include_directories(${CMAKE_SOURCE_DIR}/src/kernel/libsyscall/wrappers)
+include_directories(${CMAKE_SOURCE_DIR}/src/libc/darwin)
+include_directories(${CMAKE_CURRENT_SOURCE_DIR}/include)
+
+set(DYLIB_INSTALL_NAME "/usr/lib/system/libsystem_networkextension.dylib")
+add_darling_library(system_networkextension
+    SHARED
+        ne_session.c
+        NEHelperClient.c
+)
+target_link_libraries(system_networkextension
+    PRIVATE
+        system_c
+        system_dyld
```

Comment:
Should it not be circular? Nobody's going to link against it directly.

Reply:
Probably not circular, just reexported

Reply:
Dumb question, but what would `libsystem_networkextension` reexport?

Reply:
Nothing. Only libSystem will re-export this library.